### ISSUE TITLE
1.6.2 will not Play anything

### ISSUE THREAD

**Issue by MoonOverMiami** [jellyfin-archive/jellyfin-roku-legacy#896]
**Software Versions**
Jellyfin Server Version: 10.7.7
Roku Client Version: 1.6 build 2

**Describe the bug**

**How To Reproduce**

1. Go to any media (movie, song, etc)
2. Click on Play
3. Nothing happens.

**Expected behavior**

**Logs**

**Screenshots**

**Connection Information**
Is server local or remote? 
- Local

Is server connection http or https? 

- http

**Additional context**

**Comment by EnyoVD** [comment]
I also started having this problem tonight.

**Comment by dennisdupont** [comment]
Yep, me three

Jellyfin Server Version: 10.7.5
Roku Client Version: 1.6 build 2

**Comment by 1hitsong** [comment]
Let's start at the top. Let's see if there are any Roku logs that are helpful.

https://github.com/jellyfin/jellyfin-roku/blob/unstable/DEVGUIDE.md#bugcrash-reports

In terminal/command prompt run

`telnet rokuIPAddress 8085`

**Comment by 1hitsong** [comment]
We've found the issue. 1.6.2 expanded compatibility with HEVC and VP9 video by using newer API calls back to the Jellyfin server. These calls were added to the server back in version 10.8.1. Servers prior to this version don't know how to handle the updated calls and return a failure.

There are 2 fixes.

1. Upgrade your server instance to at least 10.8.1, but preferably latest.
2. Side load the Jellyfin on Roku client version 1.6.1 to your Roku device. We don't recommend this as this side loaded 1.6.1 client will not receive updates, but it can be used as a band-aid until you update your server instance.

Instructions to side load 1.6.1 can be found here: https://github.com/jellyfin/jellyfin-roku/discussions/897
